### PR TITLE
bump Node.js version of netlify

### PR DESCRIPTION
**Pull request by uiolee**
#2238 build failed due to low node version.

log

### CODE REVIEW

**Review by SukkaW — APPROVED** [review]

**Comment by SukkaW** [comment]
BTW, is it possible to use a string like `lts` there so that we don't have to manually bump the version?

**Comment by uiolee** [comment]
> use a string like lts

https://docs.netlify.com/configure-builds/manage-dependencies/#node-js-and-javascript

It looks like we can use any valid string that [nvm](https://github.com/nvm-sh/nvm#nvmrc) understands

**Comment by uiolee** [comment]
husky not found

https://docs.netlify.com/configure-builds/manage-dependencies/#node-js-environment
> If you set the NODE_ENV to production, any devDependencies in your package.json file will not be installed for the build.


Solution: https://typicode.github.io/husky/how-to.html#ci-server-and-docker

**Comment by SukkaW** [comment]
@uiolee Or maybe we should make the husky skip hook setup in CI. We don't commit anything inside the CI after all.

**Comment by uiolee** [comment]
@SukkaW Yes. I have commited e7f7b99599f1f096084198587ace8361cce49e4a

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build": "hexo generate",
     "eslint": "eslint .",
     "format:md": "prettier --write */**/docs/*.md */**/api/*.md",
-    "prepare": "husky"
+    "prepare": "node .husky/install.mjs"
```
Comment (SukkaW) [inline package.json:12]: @uiolee `husky` provides a first-party approach: https://typicode.github.io/husky/how-to.html#ci-server-and-docker

Let's use this instead.

**Review by SukkaW — CHANGES_REQUESTED** [review]

**Review by SukkaW — CHANGES_REQUESTED** [review]
Should we add `HUSKY=0` to Netlify as well?

https://typicode.github.io/husky/how-to.html#ci-server-and-docker

**Comment by uiolee** [comment]
adding `HUSKY=0` is acceptable but not required.

when setting `NODE_ENV=production`, all devDependencies(include `husky`) will not be installed.

when using e7f7b99599f1f096084198587ace8361cce49e4a, husky and git hooks will not be initialized。

**Review by SukkaW — APPROVED** [review]

**Pull request merged**